docs/ipam.md

```diff
@@ -65,14 +65,30 @@ peer for space:
   - it will continue to ask for space until it receives some, or its
     copy of the ring tells it all peers are full in that subnet.
 
+### Data persistence
+
+Key IPAM data is saved to disk, in a [BoltDB](https://github.com/boltdb/bolt)
+file, stored within a [data volume container](https://docs.docker.com/engine/userguide/containers/dockervolumes/#creating-and-mounting-a-data-volume-container)
+named `weavedb`.
+
+This file contains:
+
+* the division of the IP allocation range amongst peers, and
```

Comment:
Maybe it's worth mentioning that we persist more data (e.g. a local peer name), and the listed one is relevant for IPAM.